src/main/java/seedu/fridgefriend/command/RemoveCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.fridgefriend.command;
+
+import seedu.fridgefriend.food.Food;
+
+import java.util.List;
+
+
+/**
+ * Remove the food from the fridge by index specify by user.
+ */
+public class RemoveCommand extends Command {
+
+    private static final int EXTRA_INDEX = 1;
+
+    private final int indexToRemove;
+
+    public RemoveCommand(int indexToRemove) {
+        this.indexToRemove = indexToRemove - EXTRA_INDEX;
+    }
+
+    @Override
+    public void execute(List<Food> fridge) {
+        showMessageBeforeRemove(fridge);
+        removeFood(fridge);
+        showMessageAfterRemove(fridge);
+    }
+
+    private void showMessageBeforeRemove(List<Food> fridge) {
```

Comment:
If you show this message before remove action take place, this output would be weird. (There might be excpetions when you actually run remove method)